livekit-agents/livekit/agents/voice/background_audio.py

```diff
@@ -38,15 +39,25 @@ class BackgroundSound(NamedTuple):
 atexit.register(_resource_stack.close)
 
 
+class DefaultSound(enum.Enum):
+    OFFICE_AMBIENCE = "office-ambience.ogg"
+    KEYBOARD_TYPING = "keyboard-typing.ogg"
+    KEYBOARD_TYPING2 = "keyboard-typing2.ogg"
```

Comment:
Are the volumes OK?

Reply:
tweaked this slightly, to adjust volume you'd do:

```
AudioConfig(BuiltinAudioClip. OFFICE_AMBIENCE, volume=0.6)
```